docs/api-docs/apps/tutorials/sample-app-next/02-heroku-deploy.md

```diff
@@ -0,0 +1,40 @@
+# Deploy to Heroku install flow
```

Comment:
imho, this might be better here: https://developer.bigcommerce.com/api-docs/apps/quick-start.  Irrespective, this quick start will need to be updated.  Also, we might want to show how to quickly start by forking the sample app repo and using ngrok (instead of just deploying to Heroku). I only mention since we currently have a section on ngrok - https://developer.bigcommerce.com/api-docs/apps/guide/development#testing-locally-with-ngrok.

If we keep this section here, we might want to call it "Quick start: Deploy to Heroku install flow" or something to denote it's not a part of the normal tutorial flow.